rust/kernel/file_operations.rs

```diff
@@ -260,6 +260,11 @@ impl<T: FileOperations> FileOperationsVtable<T> {
         },
         write_iter: None,
     };
+
+    /// Builds an instance of [`struct file_operations`].
```

Comment:
I will take a look at these C links later in a cleanup -- some in the file are in `[]`, some aren't.

Reply:
Sounds good, thank you!